### PR TITLE
"constant" interaction force

### PR DESCRIPTION
**Pull request by Ragzouken**
closes https://github.com/IRL2/nanover-protocol/issues/54

### CODE REVIEW

File: python-libraries/nanover-core/src/nanover/imd/imd_force.py [hunk 1]
```diff
@@ -8,13 +8,23 @@
...
 
 
+class ForceCalculator(Protocol):
+    def __call__(
+        self,
+        positions: npt.NDArray,
+        interaction_position: npt.NDArray,
+        *,
+        periodic_box_lengths: Optional[npt.NDArray],
+    ) -> Tuple[float, npt.NDArray]: ...
```
Comment (Ragzouken) [inline python-libraries/nanover-core/src/nanover/imd/imd_force.py:25]: Maybe I have to change this to `**kwargs: Unpack[...]` or something to get what we really want, but either way this whole thing is exposing that some of the force calculators have parameters that are never varied in the server and aren't covered by tests as far as I can see (k and sigma).

Comment (jbarnoud) [reply python-libraries/nanover-core/src/nanover/imd/imd_force.py:25]: I think we can put `k` and `sigma` in the function body. We do not plan on exposing these variables.

File: python-libraries/nanover-core/src/nanover/imd/imd_force.py [hunk 2]
```diff
@@ -244,6 +256,26 @@ def calculate_spring_force(
     return energy, force
 
 
+def calculate_constant_force(
+    particle_position: npt.NDArray,
+    interaction_position: npt.NDArray,
+    periodic_box_lengths: Optional[npt.NDArray] = None,
+) -> Tuple[float, npt.NDArray]:
+    """
+    Applies a constant force that is independent of the distance between the particle and the interaction site.
```
[the remark this answers is not in the document]
Comment (jbarnoud) [reply python-libraries/nanover-core/src/nanover/imd/imd_force.py:265]: Missing the empty line would break the doc rendering.

**Review by jbarnoud — APPROVED** [review]
Just a tiny issue with doc rendering but otherwise, LGTM.

**Pull request merged**